hack/verify-dependencies.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
```

Comment:
Not happy with this. No. This would be just as many lines of Go. Please lets NOT add more Bash to this repo. Strong, strong no.